pkg/skaffold/docker/client.go

```diff
@@ -105,9 +128,36 @@ type ExitCoder interface {
 	ExitCode() int
 }
 
-// newMinikubeAPIClient returns a docker client using the environment variables
+// newMinikubeAPIClient returns a client using the environment variables
+// provided by minikube.
+func newMinikubeAPIClient(ctx context.Context, minikubeProfile string, minikubeDriver string) ([]string, client.CommonAPIClient, error) {
+	if minikubeDriver == noneDriver {
+		return newEnvAPIClient()
+	}
+
+	if minikubeDriver == dockerDriver {
+		return newMinikubeAPIClientWithDockerDriver(ctx, minikubeProfile)
+	}
+
+	cmd, err := cluster.GetClient().MinikubeExec(ctx, "image", "load")
```

Comment:
We don't need this method here,  ``minikube image load`` require images argument(build result).. this would just produce an error if this part get executed, we could do similar stuff here https://github.com/GoogleContainerTools/skaffold/blob/b4fba3990841eb3477f987588c5e96a34d7f8d2d/pkg/skaffold/kubernetes/loader/load.go#L88-L115 before deploying stuff to local cluster when driver is neither ``none`` or ``docker`` , but as we're currently not sure how ``minikube docker-env`` would behave for other drivers, we would prefer not including this in this pr,  what do you think?